src/validation.h

```diff
@@ -556,12 +574,21 @@ class CChainState {
     //! easily as opposed to referencing a global.
     BlockManager& m_blockman;
 
+    //! Manages the UTXO set, which is a reflection of the contents of `m_chain`.
+    std::unique_ptr<CoinsViews> m_coins_views;
+
 public:
-    CChainState(BlockManager& blockman) : m_blockman(blockman) { }
+    CChainState(
+        /* parameters forwarded to CoinsViews */
+        size_t cache_size_bytes,
+        bool in_memory,
+        bool should_wipe,
+        std::string leveldb_name = "chainstate");
```

Comment:
Hmmm is there any risk than having a default leveldb_name cause database collisions for different CChainState at some point ?

Reply:
This seems like something that would be caught pretty easily in tests if the confusion ever did happen, and I'm not sure if repeating `"chainstate"` in a few different places inline is any better. That said I'm happy to make this change if others agree it seems worthwhile.